### PR TITLE
Changed: CHANGELOG.md structure change (Issue #171).

### PR DESCRIPTION
**Pull request by Sven-Seyfert**
Hi @Danp2 , hi @mlipok 😀 .

Here is the CHANGELOG.md adjustment like I presented in issue #171 .
Our open TODOs:
- Semantic Versioning is still be to discussed
- Should we introduce a 1.0.0?

But anyways, in case you approve we will have a common "look and feel" changelog file 😎 .

Best regards
Sven

### CODE REVIEW

**Comment by Sven-Seyfert** [comment]
Damn, sorry I merged the other commits of the ReadMe suggestion into this PR, this was not intended. Please do not approve more then the changelog file.

I will revert this tomorrow, but now (23:38 in germany) I habe to sleep.

**Comment by Danp2** [comment]
@Sven-Seyfert Does this new format allow for subitems as shown in the "old" style? If so, then "new" style could look something like --

```
### Changed

- wd_demo.au3
	- Improved console output in DemoScript
	- Display screenshots in DemoWindows
	- Improved console output & in DemoAlerts
	- Corrected 'sendtext' coding in DemoAlerts
```
 instead of

```
### Changed

- Improved console output in DemoScript (wd_demo.au3)
- Display screenshots in DemoWindows (wd_demo.au3)
- Improved console output & in DemoAlerts (wd_demo.au3)
- Corrected 'sendtext' coding in DemoAlerts (wd_demo.au3)
```

Thoughts?

**Comment by Danp2** [comment]
Also, I've noticed that some of the entries are in a different layout. Which should be the preferred layout moving forward?

`- Additional error checking (_WD_GetElementFromPoint)`

or 

`- _WD_GetElementFromPoint: Additional error checking`

**Comment by Sven-Seyfert** [comment]
> Does this new format allow for subitems as shown in the "old" style? If so, then "new" style could look something like [...]

@Danp2
Of course this would be feasible. Markdown supports indentation for unordered lists. That different look came from my replacement actions that I did by RegEx. It was just more easy to reformat to the same hierarchy level. But you are right we should do it like before 😀 .

> Also, I've noticed that some of the entries are in a different layout.

Good catch Dan. This is more a personal manner of how I like it, but definitely nothing that is really important. We can stick with the information layout like you did so far.

**My question about these things is:**
Who should do these little changes? Would you expect I will do it (I would be fine with it) or it's better when you will do it to fulfill your expectations and to avoid ping-pong 🤹‍♀️ discussions?

**Comment by Sven-Seyfert** [comment]
> Damn, sorry I merged the other commits of the ReadMe suggestion into this PR, this was not intended. Please do not approve more then the changelog file.

I reverted the commit which came into this PR by accident. I will create a new branch for these changes 😉 .

**Comment by Sven-Seyfert** [comment]
@Danp2 I like that you also want to use the "Unreleased" version in the changelog. I added the comparison link to fulfill the behavior (see commit [3483025](https://github.com/Danp2/WebDriver/pull/182/commits/3483025f0969a78fbeae4d6a113ce12e602ad103)).

**Comment by Danp2** [comment]
@Sven-Seyfert I will handle the small changes as I review the PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -467,12 +467,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - _WD_DownloadFile
 - Global variable to hold session details
```
Comment (mlipok) [inline CHANGELOG.md:469]: reformating proposal
`- _WD_DownloadFile: Global variable to hold session details`

File: CHANGELOG.md [hunk 2]
```diff
@@ -441,13 +442,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- DemoUpload
+- wd_demo.au3: DemoUpload
 
 ### Changed
 
 - _WD_ElementAction handling of return status codes
```
Comment (mlipok) [inline CHANGELOG.md:449]: reformating proposal
`- _WD_ElementAction: handling of return status codes`

File: CHANGELOG.md [hunk 3]
```diff
@@ -50,25 +52,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- _WD_UpdateDriver: Check for existing 32/64 bit driver
 
 ### Changed
 
-- Sets @Extended to shown context changed (_WD_GetElementFromPoint)
-- Additional error checking (_WD_GetElementFromPoint)
+- _WD_GetElementFromPoint: Sets @Extended to shown context changed
+- _WD_GetElementFromPoint: Additional error checking
 - _WD_ExecuteScript: Add support for return of additional subnodes
 - Update various functions to use $_WD_JSON\_* constants
```
Comment (mlipok) [inline CHANGELOG.md:63]: redundant backslash ?

Comment (Danp2) [reply CHANGELOG.md:63]: Huh? You mean this? `$_WD_JSON\_* constants`

If so, I believe that @Sven-Seyfert added that in a17efcd7ae86ed63383275c0b1877049f4ab4d67.

Comment (mlipok) [reply CHANGELOG.md:63]: But why ? I do not understand it.
@Sven-Seyfert could you elaborate on this ?

File: CHANGELOG.md (hunk 1 above)
Comment (mlipok) [reply CHANGELOG.md:469]: Aha. These are two separate changes.
Ok.

File: CHANGELOG.md [hunk 4]
```diff
@@ -585,65 +585,70 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ## [0.1.0.15] - 2018-09-15
 
 ### Added
 
 - _WD_LoadWait
+- _WD_Option: Add support for BinaryFormat option
 
 ### Changed
 
 - Add support for Unicode text to _WD_ElementAction's "value" option
```
Comment (mlipok) [inline CHANGELOG.md:611]: proposal
`_WD_ElementAction: - added support for Unicode text to "value" option`

**Pull request merged**